pyscf/df/df_jk.py

```diff
@@ -330,7 +331,7 @@ def get_jk(dfobj, dm, hermi=1, with_j=True, with_k=True, direct_scf_tol=1e-13):
                      ctypes.c_int(naux), *rargs)
 
                 buf2 = lib.unpack_tril(eri1, out=buf[1])
-                vk[k] += lib.dot(buf1.reshape(-1,nao).T, buf2.reshape(-1,nao))
+                vk[k] += numpy.matmul(buf1.reshape(-1,nao).T, buf2.reshape(-1,nao))
```

Comment:
use lib.dot here. dsyrk does not match this pattern.

Reply:
done